packages/office-ui-fabric-react/src/components/Rating/Rating.base.tsx

```diff
@@ -121,7 +121,7 @@ export class RatingBase extends React.Component<IRatingProps, IRatingState> {
             role="presentation"
             type="button"
           >
-            {this._getLabel((rating as number) % 1 ? rating : i)}
+            {this._getLabel(Math.ceil(rating) === i ? rating : i)}
```

Comment:
I actually wonder if this should *always* be `i`, and the example's `ariaLabelFormat` should be changed to something like `Select {0} of {1} stars`. This label is primarily for the **button** to **select** a certain rating, and doesn't indicate anything about whether that star is currently selected (partially or fully). For example, if the current rating is 2.5 stars, clicking star 3 will set it to 3 stars, not 2.5. So the label should reflect that, not the current state.